src/server/components/annual-review/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { findListById } from "server/models/list";
+import { sendAnnualReviewCompletedEmail } from "server/services/govuk-notify";
+import { lowerCase, startCase } from "lodash";
+
+export async function sendAnnualReviewCompletedEmailForList(listId: number) {
+  const list = await findListById(listId);
+
+  if (list?.jsonData?.users) {
+    await Promise.all(
+      list.jsonData.users.map(async (postEmailAddress: string) => {
+        await sendAnnualReviewCompletedEmail(
```

Comment:
don't you want to return the promise? technically this is more of a `.forEach` at the moment. 

```
Promise.all(list.jsonData.users.map((user) => {
  return sendAnnualReviewCompletedEmail(..)
}))
```

Reply:
personally, other than logging, I don't think we need to care about this email being sent successfully.  As long as the update takes place, the Post users will see it when they login. I don't think the operation should fail if the email fails to send out, hence I don't think we need to return each promise.

Reply:
sorry, I don't mean return the Promise.all. I mean _inside_ your map method, you don't return anything. It's like doing 

```
  list.jsonData.users.map(user => { console.log(user) })
  // returns you an empty array
```

Reply:
ah sorry, yes indeed.